### PR TITLE
Implementation of switching teams

### PR DESCRIPTION
**Pull request by Ranray**

### CODE REVIEW

File: Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.SWITCH_TEAM_MESSAGE_ID, handler = SwitchTeamMessageHandler.class)
+public final class SwitchTeamMessage extends Message {
+
+    private final String username;
+    private final TeamColour team;
+    
+    public SwitchTeamMessage(String username, TeamColour team) {
```
Comment (ghost) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java:19]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        this.username = username;
+        this.team = team;
+    }
+    
+    public SwitchTeamMessage() {
```
Comment (ghost) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java:24]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java [hunk 3]
```diff
@@ -125,6 +125,22 @@ public void leaveSession(Player player, Session session) {
         controller.changeHost(name, room.getHost().getUsername());
         controller.decreaseOccupancy(name);
     }
+    
+    public void switchTeamFromPlayer(Player player) {
```
Comment (ghost) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java:129]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 4 issues
* MINOR 4 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [SessionWrapper.java#L114](https://github.com/PTS3-S34A/Game-Server/blob/62718d9ae2b8af88b3cf868da66c1cfe51f8968a/Soccar [Game-Server]/src/nl/soccar/gameserver/SessionWrapper.java#L114): Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Review by BeeHiveJava — APPROVED** [review]

**Pull request merged**